.github/workflows/docs.yml

```diff
@@ -29,18 +35,51 @@ jobs:
 
     - name: Install dependencies
       run: |
-        apt-get update && apt-get install -y rsync nodejs npm doxygen
+        apt-get update && apt-get install -y rsync doxygen curl
+        # install nvm
+        touch $HOME/.bashrc
+        curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash
+
+    - name: Install node
+      run: |
+        source $HOME/.bashrc
+        nvm install 20
+        nvm use 20
+        corepack enable
         npm install -g moxygen
 
     - name: Build docs
       run: |
+        source $HOME/.bashrc
+        nvm use 20
         qmk --verbose generate-docs
+        touch '.build/docs/.nojekyll'
+
+    - name: Set CNAME
+      if: github.repository == 'qmk/qmk_firmware'
+      run: |
+        # Override target CNAME
+        echo 'docs.qmk.fm' > .build/docs/CNAME
+
+    - name: Override CNAME
+      if: github.repository == 'tzarc/qmk_firmware'
+      run: |
+        # Temporarily override target CNAME during development
+        echo 'vitepress.qmk.fm' > .build/docs/CNAME
```

Comment:
Does this still apply or can it be removed in the final merge?

Reply:
Will be done in a follow up PR. Intentionally written such that it works on both repos.